src/node_mgmt/commands.py

```diff
@@ -212,52 +219,30 @@ def readall(s):
     return result
 
 
-def holykell_sensor_address(node):
-    from time import sleep
-    import minimalmodbus
-
-    mod = minimalmodbus.Instrument('/dev/ttyAMA0', 1, debug=True)
-    mod.serial.baudrate = 9600
+def holykell_sensor_address(node) -> dict:
+    mod = minimalmodbus.Instrument(port=DEFAULT_SERIAL_DEV, slaveaddress=1, debug=True)
+    mod.serial.baudrate = DEFAULT_SERIAL_BAUD_RATE
     mod.serial.timeout = 3
-    SLAVE_IDS_FOR_HOLYKELL = [7, 8]
     result = {}
-
     # confirm that Holykell is accessible on slave id 1
     logger.info('Checking if communication with holykell is up')
     try:
-        mod.read_registers(0, 40, 3)
+        _ = _read_holykell(mod, slave_id=1)
     except Exception as e:
         result['Error'] = f'No HPT604 detected on slave 1. Exception: {e}'
         return result
-
     # go through targeted slave_ids
     for slave_id in SLAVE_IDS_FOR_HOLYKELL:
         try:
             # check that no device already on slave address
-            mod.address = slave_id
-            mod.read_registers(0, 40, 3)
+            _ = _read_holykell(mod, slave_id=slave_id)
```

Comment:
I don't think you need '_ =` here - just calling `_read_holykell()` would suffice

Reply:
Nice! changing that